cli/ballerina-cli/src/main/java/io/ballerina/cli/utils/FileUtils.java

```diff
@@ -88,4 +98,35 @@ private static int indexOfLastSeparator(String filename) {
             return Math.max(lastUnixPos, lastWindowsPos);
         }
     }
+
+    public static String readSchema(String toolName) throws IOException {
```

Comment:
We can use Java NIO `Files.readString` to read a string

Reply:
We are reading the contents of a resource packed inside a JAR file loaded by the classloader here. Can we use `Files.readString` for such a scenario?